sql/core/src/test/resources/sql-tests/results/pgSQL/window.sql.out

```diff
@@ -0,0 +1,2247 @@
+-- !query 95 output
+1	NULL
+2	NULL
+
+
+-- !query 96
+SELECT rank() OVER (ORDER BY length('abc'))
+-- !query 96 schema
+struct<RANK() OVER (ORDER BY length(abc) ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW):int>
+-- !query 96 output
+1
+
+
+-- !query 97
+SELECT * FROM empsalary WHERE row_number() OVER (ORDER BY salary) < 10
+-- !query 97 schema
+struct<>
+-- !query 97 output
+org.apache.spark.sql.AnalysisException
+It is not allowed to use window functions inside WHERE and HAVING clauses;
+
+
+-- !query 98
+SELECT * FROM empsalary INNER JOIN tenk1 ON row_number() OVER (ORDER BY salary) < 10
+-- !query 98 schema
+struct<>
+-- !query 98 output
+org.apache.spark.sql.AnalysisException
+
+The query operator `Join` contains one or more unsupported
+expression types Aggregate, Window or Generate.
+Invalid expressions: [row_number() OVER (ORDER BY default.empsalary.`salary` ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW)];
+
+
+-- !query 99
+SELECT rank() OVER (ORDER BY 1), count(*) FROM empsalary GROUP BY 1
+-- !query 99 schema
+struct<>
+-- !query 99 output
+org.apache.spark.sql.AnalysisException
+
+The query operator `Aggregate` contains one or more unsupported
+expression types Aggregate, Window or Generate.
+Invalid expressions: [RANK() OVER (ORDER BY 1 ASC NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW)];
+
+
+-- !query 100
+SELECT * FROM rank() OVER (ORDER BY random())
+-- !query 100 schema
+struct<>
+-- !query 100 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+no viable alternative at input 'ORDER'(line 1, pos 27)
+
+== SQL ==
+SELECT * FROM rank() OVER (ORDER BY random())
+---------------------------^^^
+
+
+-- !query 101
+SELECT * FROM empsalary WHERE (rank() OVER (ORDER BY random())) > 10
+-- !query 101 schema
+struct<>
+-- !query 101 output
+org.apache.spark.sql.AnalysisException
+It is not allowed to use window functions inside WHERE and HAVING clauses;
+
+
+-- !query 102
+SELECT * FROM empsalary WHERE rank() OVER (ORDER BY random())
+-- !query 102 schema
+struct<>
+-- !query 102 output
+org.apache.spark.sql.AnalysisException
+It is not allowed to use window functions inside WHERE and HAVING clauses;
+
+
+-- !query 103
+select rank() OVER (PARTITION BY four, ORDER BY ten) FROM tenk1
+-- !query 103 schema
+struct<>
+-- !query 103 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+no viable alternative at input 'ORDER'(line 1, pos 39)
+
+== SQL ==
+select rank() OVER (PARTITION BY four, ORDER BY ten) FROM tenk1
+---------------------------------------^^^
+
+
+-- !query 104
+SELECT range(1, 100) OVER () FROM empsalary
+-- !query 104 schema
+struct<>
+-- !query 104 output
+org.apache.spark.sql.AnalysisException
+Undefined function: 'range'. This function is neither a registered temporary function nor a permanent function registered in the database 'default'.; line 1 pos 7
```

Comment:
This error is not expected one: https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/expected/window.out#L2982

Reply:
Although the message is misleading, I think that they represent the same: there's no existent `range` aggregated function. I should JIRA that?